exasol/analytics/query_handler/context/proxy/bucketfs_location_proxy.py

```diff
@@ -1,6 +1,6 @@
 import logging
 
-import exasol.bucketfs as bfs
+import exasol.bucketfs as bfs  # type: ignore[import-untyped]
```

Comment:
I think, we can do this in the pyproject.toml for all imports, but we probably should add py.typed to bucketfs-python

Reply:
See last push